Saucier720-app/src/app/login/FEC/login-form/login-form.component.html

```diff
@@ -5,6 +5,10 @@
                 <span class="logo"></span>
             </div>
         </div>
+        <div class="error-msg" *ngIf="failedLogin">
+            <i class="fa fa-exclamation-circle"></i>
+            <p>Invalid Login</p>
```

Comment:
Do you think it would be possible to specify if it was a username or password error? Im not sure how easy it is but 'Invalid Login' can be annoying so maybe we want to specify if an account already exists and the password is incorrect

Reply:
Not until the database rework, and also this would require better error handling on backend, and changing the way we validate users, because right now it either returns data or doesn't, no specifics. So maybe something we can do in the future.